src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -0,0 +1,482 @@
+                                        }}
+                                    >
+                                        {actionOptions.map((actionType) =>
+                                            (<MenuItem key={actionType} value={actionType}>
+                                                {actionType}
+                                            </MenuItem>)
+                                        )}
+                                    </Select>
+                                </FormControl>
+                            </Grid>
+                        </TableCell>
+                        <TableCell style={{ width: 115 }}>
+                            <Grid container>
+                                <Grid>
+                                    Object
+                                </Grid>
+                                <Grid>
+                                    <SwapVertIcon onClick={handleObjectSort} style={{ color: "grey" }} />
+                                </Grid>
+                            </Grid>
+                            <Grid>
+                                <FormControl
+                                    variant="outlined"
+                                >
+                                    <InputLabel shrink={false} style={{ lineHeight: "5px" }} margin="dense" >{objectType ? "" : "Object Type"}</InputLabel>
+                                    <Select
+                                        id="select"
+                                        labelId="label"
+                                        value={objectType}
+                                        onChange={handleObjectSelection}
+                                        style={{ width: 130 }}
+                                        classes={{
+                                            outlined: classes.outlined,
+                                            root: classes.root,
+                                            "MuiInputBase-input": classes.MuiInputBase
+                                        }}
+                                    >
+                                        {objectOptions.map((objectType) =>
+                                            (<MenuItem key={objectType} value={objectType}>
+                                                {objectType}
+                                            </MenuItem>)
+                                        )}
+                                    </Select>
+                                </FormControl>
+                            </Grid>
+                        </TableCell>
+                        <TableCell>
+                            <Grid container>
+                                <Grid>
+                                    Details
+                                </Grid>
+                            </Grid>
+                            <Grid>
+                                <br />
+                            </Grid>
```

Comment:
what's the point of this? if you need more space, either consolidate the `<br/>` tag with the "Details" text, or use CSS to create more margin/padding